### DISCUSSION TITLE
What does "desktops running on the VM" mean?

### DISCUSSION THREAD

**Discussion by kennethjyang** [isar/isar#1353] [Help] [answered]
Hi,
I'm relatively new to Flutter and want to use Isar for a cross-platform project. I noticed on the limitations page Isar can only work on a "desktop running on the VM". What does this mean? My interpretation is that this means I can't make a native desktop app that uses Isar.

Thanks for clarifying!

**Comment by nobkd** [comment, marked as answer]
I think it means, that it is supported for devices that are able to run the Dart VM

Here's the indroductory image of [dart.dev](https://dart.dev):

![Paint your ui to live with the Dart VM](https://dart.dev/assets/dash/2x/paint-your-ui.png)

**Reply by kennethjyang** [reply]
Right, so it'll work for development on desktop targets, but does this also mean it will work for a compiled native desktop application?

**Reply by nobkd** [reply]
Wouldn't make sense otherwise, right?

**Reply by kennethjyang** [reply]
I would hope so 😅 I just got nervous when I saw the "VM" bit and thought it wasn't natively compatible
